### PR TITLE
fix: wording and grammar

### PR DESCRIPTION
**Pull request by Gitleptune**
- fix wording and grammar in readme.md and contributing.md

sorry for taking a while to open pr

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -30,7 +34,7 @@ Tapetti is a wallpaper manager that you never asked for. Feature-rich yet minima
 
 </div>
 
-# Featuers
+## Features
```
Comment (Muhimen123) [inline README.md:37]: Why did you add a extra `#` though? 😅

**Review by Muhimen123 — APPROVED** [review]
Looks good to me

File: README.md (hunk 1 above)
Comment (Gitleptune) [reply README.md:37]: It's usually good practice to remain the `#`/`<h1>` for titles for search engines and stuff, I don't think it matters for github but it's still better practice to use more specific headings.

Comment (Muhimen123) [reply README.md:37]: I see.
